src/Bicep.Core.IntegrationTests/Scenarios/ResourceListFunctionTests.cs

```diff
@@ -32,7 +33,8 @@ public void List_wildcard_function_on_resource_references()
 })
 ");
 
-            result.Should().NotHaveAnyDiagnostics();
+            // Ignore outputs-should-not-contain-secrets warnings which are expected
+            result.Diagnostics.Where(d => d.Code != "outputs-should-not-contain-secrets").Should().BeEmpty();
```

Comment:
Now that we have `#disable-next-line` - could we just disable this in the source text rather than ignoring it here?

Reply:
Done